src/ElementQueries.js

```diff
@@ -12,7 +12,7 @@
     } else {
         root.ElementQueries = factory(root.ResizeSensor);
     }
-}(this, function (ResizeSensor) {
+}(typeof window !== 'undefined' ? window : this, function (ResizeSensor) {
```

Comment:
I've to confess that I'm pretty new to JavaScript, so I don't exactly know why this isn't working while building my app. But since ResizeSensor isn't the only lib I use, I compared your mechanism with the one of [Highcharts](https://code.highcharts.com/highcharts.src.js) which is working fine in our build process.

If you still need more information don't hesitate to bother me ;)